compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h

```diff
@@ -26,6 +27,11 @@ using TileSizesListTypeRef = ArrayRef<SmallVector<int64_t>>;
 /// Typedef for scalable tile flags at different levels of tiling.
 using ScalableTileFlagsListType = SmallVector<SmallVector<bool>>;
 using ScalableTileFlagsListTypeRef = ArrayRef<SmallVector<bool>>;
+/// Flag to add attributes for tuner.
+inline llvm::cl::opt<bool>
+    clSetTunerAttr("iree-config-add-tuner-attributes",
+                   llvm::cl::desc("add attributes for tuner"),
```

Comment:
nit: It is a description showing up on CLI, so we should complete the sentence.

```suggestion
                   llvm::cl::desc("Add attributes for tuner."),
```